src/selectionHandler.ts

```diff
@@ -127,10 +129,10 @@ async function handleImports(editor: TextEditor, completion: any) {
         completionSelection,
         CodeActionKind.QuickFix
       );
-      let importCommand = findImports(codeActionCommands)[0];
+      let importCommand = findImports(codeActionCommands!)[0];
 
-      if (importCommand) {
-        await workspace.applyEdit(importCommand.edit);
+      if (importCommand.edit) {
```

Comment:
are you sure this is correct, what would be returned from findImports in case of no import command?